code/modules/mob/language/station.dm

```diff
@@ -85,6 +85,20 @@
 	flags = WHITELISTED
 	syllables = list("qr","qrr","xuq","qil","quum","xuqm","vol","xrim","zaoo","qu-uu","qix","qoo","zix")
 
+/datum/language/skrellfar
```

Comment:
Doesn't look like Skrell can actually get this, is that intentional?

Reply:
yeah. whitelisted for Skrell too, possibly subject to change, but my Plan with the Far Kingdoms does involve putting a lot more trust into players to portray it correctly than I expect from current Near Kingdom Skrell.